audit/projects/kubernetes-public/iam.json

```diff
@@ -116,6 +116,7 @@
     {
       "members": [
         "group:gke-security-groups@kubernetes.io",
+        "serviceAccount:gke-nodes-aaa@kubernetes-public.iam.gserviceaccount.com",
```

Comment:
This is the result of running `terraform apply` for `clusters/projects/kubernetes-public/aaa` using the last commit of https://github.com/kubernetes/k8s.io/pull/1974.  There was an authoritative `google_iam_policy` terraform resource that kept overwriting other non-authoritative `add-iam-policy-binding` equivalents in terraform or bash.

I opted to move the project level bindings for accounts/groups not managed by terraform over to `ensure-main-project.sh`, and re-ran `terraform apply`... these are all of the bindings that should be present